iset.mm

```diff
@@ -154365,6 +154365,517 @@ the additive structure must be abelian (see ~ ringcom ), care must be
+      UDZUAZOZUKUPAMZKASJDSURUKUSJKDAUKUMDMUNAMUSUMUOCDAEUNFGUOTZHUBUEUCJKDAUPA
+      UQUQTUFUGUKULABUQJKABUOCDLEFGHIUTUHUIUJ $.
+  $}
+
+  ${
+    lmodvsdi.v $e |- V = ( Base ` W ) $.
+    lmodvsdi.a $e |- .+ = ( +g ` W ) $.
+    lmodvsdi.f $e |- F = ( Scalar ` W ) $.
+    lmodvsdi.s $e |- .x. = ( .s ` W ) $.
+    lmodvsdi.k $e |- K = ( Base ` F ) $.
+    $( Distributive law for scalar product (left-distributivity).  (Contributed
+       by NM, 10-Jan-2014.)  (Revised by Mario Carneiro, 22-Sep-2015.) $)
+    lmodvsdi $p |- ( ( W e. LMod /\ ( R e. K /\ X e. V /\ Y e. V ) ) ->
+                     ( R .x. ( X .+ Y ) ) = ( ( R .x. X ) .+ ( R .x. Y ) ) ) $=
+      ( wcel co wceq wa cfv eqid clmod wi w3a cplusg cur lmodlema simpld simp2d
+      cmulr 3expia anabsan2 exp4b com34 3imp2 ) GUAOZBEOZHFOZIFOZBHIAPCPBHCPZBI
+      CPAPQZUOUPURUQUTUOUPURUQUTUOUPURUQRZUTUBUOUPUPRZVAUTUOVBVAUCZUSFOZUTBBDUD
+      SZPHCPUSUSAPQZVCVDUTVFUCBBDUISZPHCPBUSCPQDUESZHCPHQRAVEBBCVGVHDEFGIHJKMLN
+      VETVGTVHTUFUGUHUJUKULUMUN $.
+  $}
+
+  ${
+    lmodvsdir.v $e |- V = ( Base ` W ) $.
+    lmodvsdir.a $e |- .+ = ( +g ` W ) $.
+    lmodvsdir.f $e |- F = ( Scalar ` W ) $.
+    lmodvsdir.s $e |- .x. = ( .s ` W ) $.
+    lmodvsdir.k $e |- K = ( Base ` F ) $.
+    lmodvsdir.p $e |- .+^ = ( +g ` F ) $.
+    $( Distributive law for scalar product (right-distributivity).
+       (Contributed by NM, 10-Jan-2014.)  (Revised by Mario Carneiro,
+       22-Sep-2015.) $)
+    lmodvsdir $p |- ( ( W e. LMod /\ ( Q e. K /\ R e. K /\ X e. V ) ) ->
+                    ( ( Q .+^ R ) .x. X ) = ( ( Q .x. X ) .+ ( R .x. X ) ) ) $=
+      ( wcel co wceq wa clmod w3a cmulr cfv cur lmodlema simpld simp3d anabsan2
+      eqid 3expa exp42 3imp2 ) IUAQZCGQZDGQZJHQZCDBRJERCJERDJERZARSZUNUOUPUQUSU
+      NUOUPTZTUQUSUNUTUQUQTZUSUNUTVAUBZURHQZDJJARERURURARSZUSVBVCVDUSUBCDFUCUDZ
+      RJERCURERSFUEUDZJERJSTABCDEVEVFFGHIJJKLNMOPVEUJVFUJUFUGUHUKUIULUM $.
+  $}
+
+  ${
+    lmodvsass.v $e |- V = ( Base ` W ) $.
+    lmodvsass.f $e |- F = ( Scalar ` W ) $.
+    lmodvsass.s $e |- .x. = ( .s ` W ) $.
+    lmodvsass.k $e |- K = ( Base ` F ) $.
+    lmodvsass.t $e |- .X. = ( .r ` F ) $.
+    $( Associative law for scalar product.  (Contributed by NM, 10-Jan-2014.)
+       (Revised by Mario Carneiro, 22-Sep-2015.) $)
+    lmodvsass $p |- ( ( W e. LMod /\ ( Q e. K /\ R e. K /\ X e. V ) ) ->
+                      ( ( Q .X. R ) .x. X ) = ( Q .x. ( R .x. X ) ) ) $=
+      ( wcel co wceq wa cfv eqid clmod w3a cplusg cur lmodlema simprld anabsan2
+      3expa exp42 3imp2 ) HUAOZAFOZBFOZIGOZABDPICPABICPZCPQZUKULUMUNUPUKULUMRZR
+      UNUPUKUQUNUNRZUPUKUQURUBUOGOBIIHUCSZPCPUOUOUSPQABEUCSZPICPAICPUOUSPQUBUPE
+      UDSZICPIQUSUTABCDVAEFGHIIJUSTLKMUTTNVATUEUFUHUGUIUJ $.
+  $}
+
+  ${
+    lmod0cl.f $e |- F = ( Scalar ` W ) $.
+    lmod0cl.k $e |- K = ( Base ` F ) $.
+    lmod0cl.z $e |- .0. = ( 0g ` F ) $.
+    $( The ring zero in a left module belongs to the ring base set.
```

Comment:
Better: "The ring zero in a left module belongs to the set of scalars.", Also for the next theorem, and in set.mm.

Reply:
Changed.